src/views/dialogs/DepositDialog2/consts.ts

```diff
@@ -0,0 +1,3 @@
+export const DYDX_DEPOSIT_CHAIN = 'dydx-mainnet-1';
+export const DYDX_CHAIN_USDC_DENOM =
+  'ibc/8E27BA2D5493AF5636760E354E46004562C46AB7EC0CC4C1CA14E9E20E2545B5';
```

Comment:
I believe this exists in `useTokenConfigs`, would just need a check for chain-id. Not opposed to having in constants/token either

Reply:
yaaaa im lazy and this is better as a constant i think 😅